src/ansys/aedt/core/extensions/maxwell3d/create_coil.py

```diff
@@ -105,319 +91,108 @@ def __init__(self, withdraw: bool = False):
-        inner_distance_description = tk.Text(
-            self.root, width=40, height=2, name="inner_distance_description", wrap=tk.WORD
-        )
-        inner_distance_description.insert(
-            "1.0", "Distance between the coil and the inner rectangle (length along X or Y axis)"
-        )
-        inner_distance_description.grid(row=8, column=2, **DEFAULT_PADDING)
-        inner_distance_description.config(state=tk.DISABLED)
-
-        arc_segmentation = ttk.Label(self.root, text="Arc segmentation:", style="PyAEDT.TLabel", width=20)
-        arc_segmentation.grid(row=9, column=0, pady=5)
-        self.__widget["arc_segmentation_text"] = tk.Text(self.root, width=20, height=1, name="arc_segmentation")
-        self.__widget["arc_segmentation_text"].insert(tk.END, "4")
-        self.__widget["arc_segmentation_text"].configure(
+        self.__widget["is_vertical"].var = is_vertical
+        self.__widget["is_vertical"].grid(row=row, column=1, sticky="", padx=5)
+        row += 1
+        return row
+
+    def _add_parameter_row(self, tab, parameter, default_value, row):
+        widget_name = parameter.replace(" ", "_").lower()
+        parameter_label = ttk.Label(tab, text=parameter, style="PyAEDT.TLabel", width=20)
+        parameter_label.grid(row=row, column=0, **DEFAULT_PADDING)
+        self.__widget[widget_name] = tk.Text(tab, width=20, height=1, name=widget_name)
+        self.__widget[widget_name].configure(
             bg=self.theme.light["pane_bg"], foreground=self.theme.light["text"], font=self.theme.default_font
         )
-        self.__widget["arc_segmentation_text"].grid(row=9, column=1, **DEFAULT_PADDING)
-        arc_segmentation_description = tk.Text(
-            self.root, width=40, height=2, name="arc_segmentation_description", wrap=tk.WORD
-        )
-        arc_segmentation_description.insert("1.0", "number of segments into which to divide the coil corners")
-        arc_segmentation_description.grid(row=9, column=2, **DEFAULT_PADDING)
-        arc_segmentation_description.config(state=tk.DISABLED)
-
-        section_segmentation = ttk.Label(self.root, text="Section segmentation:", style="PyAEDT.TLabel", width=20)
-        section_segmentation.grid(row=10, column=0, pady=5)
-        self.__widget["section_segmentation_text"] = tk.Text(self.root, width=20, height=1, name="section_segmentation")
-        self.__widget["section_segmentation_text"].insert(tk.END, "6")
-        self.__widget["section_segmentation_text"].configure(
-            bg=self.theme.light["pane_bg"], foreground=self.theme.light["text"], font=self.theme.default_font
+        self.__widget[widget_name].grid(row=row, column=1, **DEFAULT_PADDING)
+        self.__widget[widget_name].insert("1.0", f"{default_value}")
+        row += 1
+        return row
+
+    def _add_export_button(self, tab, row):
+        export_points_button = ttk.Button(
+            tab, text="Parameters", command=self.show_pictures_popup, width=10, style="PyAEDT.TButton"
```

Comment:
```suggestion
            tab, text="Parameters help", command=self.show_pictures_popup, width=10, style="PyAEDT.TButton"
```
I would probably modify button text to this